tests/ui/resolve/open-ns-2.rs

```diff
@@ -0,0 +1,19 @@
+//@ build-pass
+//@ aux-crate: my_api=open-ns-my_api.rs
+//@ aux-crate: my_api::utils=open-ns-my_api_utils.rs
+//@ aux-crate: my_api::core=open-ns-my_api_core.rs
+//@ compile-flags: -Z namespaced-crates
+//@ edition: 2024
+
+//@ ignore-test FIXME(packages_as_namespaces): feature not implemented yet
+
+use my_api::core::{core_fn, core_fn2};
+use my_api::utils::*;
+use my_api::*;
```

Comment:
In the 3-scope model this would import from both `my_api(crate)` and `my_api(open_ns)`.
Therefore it will introduce names `utils(mod)`, `root_function` and `core` into this module, but not `utils(crate)` because it would be shadowed by `utils(mod)`.
Need to test this behavior.